routes.py

```diff
@@ -8,7 +8,8 @@
 session = database_handler.connect_to_database()
 
 
-@obscura.route('/player/', methods = ['GET'])
+# @obscura.route('/player/', methods = ['GET'])
```

Comment:
What's this comment for?

Reply:
i have changed url in new commit that i am going to commit so i commented the old ones
to make changes in frontend

Reply:
Going to? You already committed, and we can see what you changed in git diff, so we can remove the comment